presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraConnector.java

```diff
@@ -94,6 +93,8 @@ public class TestCassandraConnector
             ImmutableSet.of(),
             Optional.empty(),
             ImmutableMap.of());
+
+    private CassandraServer server;
```

Comment:
Move this private member to the private section below.

Reply:
Updated